src/cmdlib.sh

```diff
@@ -462,12 +478,23 @@ EOF
     else
         rm -vf "${local_overrides_lockfile}"
     fi
+    
+    contentset_paths=()
     if [ -e "${configdir}/content_sets.yaml" ]; then
+        contentset_paths+=( "${configdir}/content_sets.yaml" )
+    fi
+    if [ -e "${workdir}/src/extrepos/content_sets.yaml" ]; then
+        contentset_paths+=( "${workdir}/src/extrepos/content_sets.yaml" )
+    fi
```

Comment:
This doesn't seem correct. We should be calling `create_content_manifest` just once. Let's look for it first in the src config repo, then fallback to looking in the extrepos.